README.md

```diff
@@ -83,6 +83,16 @@ The YAML plugin supports you with filling out the document. When hitting Ctrl +
 
 ![Validation in Visual Studio Code](./doc/vs-code-validation.png)
 
+To use the YAML plugin correctly, you need to assign the schema to the plugin. To do this, open the command palette (**Shift+Command+P** on Mac or **Ctrl+Shift+P** for Windows/Linux) and search for **Preferences: Open Workspace Settings (JSON)**. Then add the following lines:
+
+```json
+{
+  "yaml.schemas": {
+          "https://schema.thethings.network/devicerepository/1/schema": "vendor/**/*.yaml"
+      }
+}
+```
```

Comment:
This assumes that the file is empty. Also indentation isn't correct.


```suggestion
To use the YAML plugin correctly, you need to assign the schema to the plugin. To do this, open the command palette (**Shift+Command+P** on Mac or **Ctrl+Shift+P** for Windows/Linux) and search for **Preferences: Open Workspace Settings (JSON)**. Then add the following lines:

```json
{
  // other settings go here
  
  "yaml.schemas": {
    "https://schema.thethings.network/devicerepository/1/schema": "vendor/**/*.yaml"
  }
}
```
```

Finally do we reference the YAML extension? It's not part of VS Code, it needs to be installed.

Reply:
Read the lines above, there we reference the yaml extension: https://github.com/TheThingsNetwork/lorawan-devices/blob/fafc80e1a1bfaed7db66b2954005115196bed106/README.md?plain=1#L80-L82